portal-api/src/v1/routes.js

```diff
@@ -44,32 +47,43 @@ openRouter.route('/users/reset-password/:resetPwdToken').post(users.resetPasswor
 
 // API Key Routes
 openRouter.route('/feedback').post(checkApiKey, feedback.create);
-openRouter.route('/user').post(checkApiKey, users.create); // This endpoint is only used by mock-data-loader, for setting up an initial user
+// This endpoint is only used by mock-data-loader, for setting up an initial user
+openRouter.route('/user').post(checkApiKey, users.create);
+
+/**
+ * Mandatory Criteria routes
+ * Allow POST & PUT of MC HTML tags
+ * on non-production environments only
+ */
+authRouter
+  .route('/mandatory-criteria')
+  .post(validate({ role: ['editor'] }), mandatoryCriteria.create);
 
-const authRouterAllowXss = express.Router();
-authRouterAllowXss.use(passport.authenticate('jwt', { session: false }));
+authRouter
+  .route('/mandatory-criteria/:version')
+  .put(validate({ role: ['editor'] }), mandatoryCriteria.update);
 
-authRouterAllowXss.route('/users').get(users.list).post(users.create);
-authRouterAllowXss.route('/users/:_id').get(users.findById).put(users.updateById).delete(users.remove);
-authRouterAllowXss.route('/users/:_id/disable').delete(users.disable);
+// Enable XSS
+authRouter.use(cleanXss);
```

Comment:
Do we know why we need this?

Reply:
Previously we had it enabled only for a few endpoints. Now we have it enabled for all of them